spec/cucumber/formatter/http_io_spec.rb

```diff
@@ -6,15 +6,15 @@
 require 'spec_helper'
 require 'cucumber/formatter/io'
 
-RSpec.shared_context 'an HTTP server accepting file requests' do
-  module WEBrick
-    module HTTPServlet
-      class ProcHandler < AbstractServlet
-        alias do_PUT do_GET # Webrick #mount_proc only works with GET,HEAD,POST,OPTIONS by default
-      end
+module WEBrick
```

Comment:
Same here but with Class.new (avoids rspec leakage)

Reply:
Same: it was actually already leaking to everything. The change has just made it explicit